docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+     - Estimate transition rates.
+   * - `Disability Weights`_
+     - Proportion of full health not experienced due to disability associated
+       with a given condition.
+     - Measure disability attributed to cause model states.
+   * - `Cause-specific Mortality`_
+     -
+     -
+   * - `Excess Mortality`_
+     -
+     -
 
-Incidence
-+++++++++
+Prevalence
+++++++++++
+
+Prevalence is defined as the **proportion of a given population that possesses
+a given condition or trait** at a given time-point.
 
-Birth prevalence
+  For example, the prevalence of obesity in the United States was
+  approximately 40% in 2016.
+
+When a *time-frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a
+*time-point* (such as 1/1/16) is reported, it is commonly assumed that the
+reported prevalence represents the prevalence of the *midpoint* of
+that time-frame (7/1/16 is the midpoint for the time frame of all of 2016).
+However, this may not always be the case and it should be considered when
+relevant.
+
+Prevalence data can be used to **initialize cause model states** and
+represents the **probability that a simulant will begin the simulation in a
+given state.**
+
+  For example, the probability that a simulant in a model of obesity in the
+  United States beginning in 2016 will begin the simulation as obese is 0.4 or
+  40%.
+
+Birth Prevalence
 ++++++++++++++++
 
-Remission
+Birth prevalence is defined as the **proportion of live births in a given
+population that possess a given condition or trait at birth.**
+
+  For example, the birth prevalence for cleft lip in the United States in 2006
+  was 10.6 per 10,000 live births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model
+states** and represent the **probability that a simulant who is born during
+the simulation will be born into a given neonatal cause model state.**
+
+  For example, the probability that a simulant born during a simulation of
+  cleft lip in the United States in 2006 is 0.00106, or 0.106%.
+
+Incidence
 +++++++++
 
-Prevalence
+Incidence rates are defined as the **number of new cases of a condition that
+occur per person-year of the at-risk population (individuals without
+condition).** Specifically, the at-risk population can be represented as
+`1 - condition prevalence`.
+
+  For example, the incidence of multiple sclerosis (MS) in the United States
+  is 2.8 per 100,000 person-years of the at-risk population.
+
+  This suggests that if we followed 100,000 individuals without MS for 1
+  year each (100,000 people * 1 year = 100,000 person-years), we would expect
+  2.8 of these individuals to develop MS within this timeframe.
+
+  Alternatively, if we followed 50,000 individuals without MS for 2 years each
+  (50,000 people * 2 years = 100,000 person-years), we also would expect 2.8
+  of these individuals to develop MS within this timeframe.
+
+Incidence can be used to **estimate cause model transition rates** and can
```

Comment:
Be consistent about incidence vs hazard vs transition etc.

Reply:
I read through an introduction to survival analysis, and if I am understanding things correctly, all three of these are different terms for exactly the same thing, just in different contexts or coming from different disciplines:

incidence rate (epidemiology) = hazard rate (survival analysis) = transition rate (Markov processes) = force of mortality (demographics) = failure rate (reliability engineering)

Namely, all of these are the negative of the logarithmic derivative of the survival function S(t) = complementary cumulative distribution function for the random variable T representing the time until the relevant event happens:

S(t) = P(T >t)
lambda(t) = - d/dt (log S(t)) = -S'(t)/S(t)

The question then becomes how to estimate lambda(t) from data.

Does that seem accurate?

And yes, we should be consistent with terminology. I just wanted to make sure I understand what we're talking about.

Reply:
You're accurate, but we're trying to bridge GBD terminology (incidence as essentially cumulative incidence over the year in an age group) with our transition rates.